app/components/post-title.js

```diff
@@ -93,15 +93,9 @@ export default Ember.Component.extend({
       let newTitle = this.get('newTitle');
 
       post.set('title', newTitle);
-      post.save().then(() => {
+      this.get('doSave')(post).then(() => {
```

Comment:
this is currently resulting in a few errors that cause some post editing acceptance tests to fail. This is because you aren't actually returning the promise from the save call, so `this.get('doSave')(post)` returns `undefined` when you really want it to return a promise.